src/parakh/Shop.java

```diff
@@ -0,0 +1,102 @@
+class Shop {
+
+	int stockofMaggie=50;
+	int stockofDosa=43;
+	int stockofOilPouches=39;
+	int stockofPanipuri=43;
+	int stockofmasala=73;
+
+	void requestedQ(boolean reqType,int maggieq, int dosaQ, int oilQ, int panipuriQ, int masalaQ ){ 
+		if(maggieq==0) 
```

Comment:
I guess you have put condition for, If customer pass 0 then you will saying that stock is not available but instead of this you can print something else which gives more idea when customer request for 0 quantity. , because this OOS message convey wrong, don’t you think so?